First/src/MediumDifficultyAssignmnets/AnnualSalary.java

```diff
@@ -0,0 +1,57 @@
+package firstPackage.First.src.MediumDifficultyAssignmnets;
+
+public class AnnualSalary {
+    private float tax;
+
+    public float IncomeTax(float income) {
+        int slab = (income < 500000) ? 0 : (500000 <= income && income <= 700000)
+                ? 10 : (700000 <= income && income <= 1000000) ? 15 : (1000000 < income && income >= 1500000) ? 30 : 20;
```

Comment:
please correct the conditions as per problem statements

Reply:
Done!